opamp/observiq/observiq_client.go

```diff
@@ -430,6 +430,7 @@ func (c *Client) createPackageMaps(
 func (c *Client) installPackageFromFile(file *protobufs.DownloadableFile, curPackageStatuses *protobufs.PackageStatuses) {
 	// There should be no reason for us to exit this function unless we detected a problem with the Updater's installation
 	defer c.safeSetUpdatingPackage(false)
+	defer c.downloadableFileManager.CleanupArtifacts()
```

Comment:
This seems like this will always trigger do we want that?

Reply:
reverted this change, thought it made sense because of the comment above but it apparently breaks things.